civicaseextras.php

```diff
@@ -134,8 +136,45 @@ function civicaseextras_civicrm_entityTypes(&$entityTypes) {
   _civicaseextras_civix_civicrm_entityTypes($entityTypes);
 }
 
-function civicaseextras_civicrm_alterAngular(\Civi\Angular\Manager $angular) {
-  _civicaseextras_civicrm_alterAngular($angular);
+/**
+ * Implements hook_civicrm_alterAngular().
+ */
+function civicaseextras_civicrm_alterAngular(AngularManager $angular) {
+  $caseDurationCustomField = CRM_Civicaseextras_Services_CustomValueService::getCustomField('case_stats', 'duration');
```

Comment:
Sorry to ask for further alteration, but what do you think about making the `CustomValueService` metho non-static and instead of injecting the custom field into the `CaseDuration` class injecting an instance of `CustomValueService`. Then we're actually close to dependency injection and if we want to test this class it should be a lot easier.